packages/utils/src/scripts/gecko/custom-tokens.json

```diff
@@ -13,5 +13,25 @@
     "id": "danchortoken",
     "symbol": "usn",
     "name": "USN"
+  },
+  {
+    "id": "chexchexchex",
+    "symbol": "chex",
+    "name": "Chintai"
+  },
+  {
+    "id": "eoshashcoins",
+    "symbol": "hash",
+    "name": "Hash"
+  },
+  {
+    "id": "eoshashcoins",
+    "symbol": "hpoint",
+    "name": "HPoint"
+  },
+  {
+    "id": "ram.defi",
+    "symbol": "bram",
+    "name": "bRAM"
```

Comment:
the `gecko-coins.json` already has raw metadata for this token. 

```json
{
    "id": "defibox-bram",
    "symbol": "bram",
    "name": "Defibox bRAM"
  }
```

we don't need to add it to the `custom-tokens.json`